### ISSUE TITLE
iCal file or Web Calendar file?

### ISSUE THREAD

**Issue by jrcharney** [OpenDataSTL/Build4STL#1]
Hey Jon,

Just a suggestion.  Not sure if the Event Schedule file was saved from last year, but hopefully this git repo has it folded away somewhere in it's history.  It would be good to hold on to the old stuff to remember what we did back then.

Any way, for the Event Schedule file (Event Schedule 2015 and onward) perhaps an .ical file or web cal file would be ideal.  Though I'm the kind of person who likes to use Google Calendar.

An itenerary was a good idea, though splitting it up week by week seems a bit much.  IDK, what do you think?

**Comment by jonathanleek** [comment]
Yeah, we'll fold it into a google calendar file once we have stuff
finalized; what we have in github right now is just a rough outline for
planning purposes.

On Fri, Feb 20, 2015 at 12:00 PM, Jason Charney <notifications@github.com>
wrote:

> Hey Jon,
>
> Just a suggestion. Not sure if the Event Schedule file was saved from last
> year, but hopefully this git repo has it folded away somewhere in it's
> history. It would be good to hold on to the old stuff to remember what we
> did back then.
>
> Any way, for the Event Schedule file (Event Schedule 2015 and onward)
> perhaps an .ical file or web cal file would be ideal. Though I'm the kind
> of person who likes to use Google Calendar.
>
> An itenerary was a good idea, though splitting it up week by week seems a
> bit much. IDK, what do you think?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenDataSTL/Build4STL/issues/1>.
>



-- 
Jonathan Leek
